spiceaidocs/docs/components/data-connectors/abfs.md

```diff
@@ -18,36 +15,43 @@ datasets:
   - from: abfs://foocontainer/taxi_sample.csv
     name: azure_test
     params:
-      azure_account: spiceadls
-      azure_access_key: abc123==
+      abfs_account: spiceadls
+      abfs_access_key: abc123==
       file_format: csv
 ```
 
-## Dataset Schema Reference
+## Configuration
 
 ### `from`
 
-The ABFS-compatible URI to a folder or object in one of two forms:
+Defines the ABFS-compatible URI to a folder or object:
 
 - `from: abfs://<container>/<path>` with the account name configured using `abfs_account` parameter, or 
 - `from: abfs://<container>@<account_name>.dfs.core.windows.net/<path>`
 
 :::note
 
-A valid URI must always be specified in the `from` field, even if you are setting the account or container name using [secrets](/components/secret-stores/index.md). When using secrets use a dummy account/container name and the values will be replaced with the values contained by the secrets at runtime.
+A valid URI must always be specified in the `from` field, even if you are setting the account or container name using [secrets](/components/secret-stores/index.md). When using secrets, a dummy account/container name must be used. The values will be replaced at runtime with the values contained in the secrets.
```

Comment:
Similar to my other note, we shouldn't allow specifying the account/container name in two places. Let's pick just one and allow the secret replacement syntax to work in that place.

Reply:
Fixed as discussed